src/main/resources/db/changelog/changes/03-create-user-role-table.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-user-role-table
+      author: artemreizer
+      changes:
+        - createTable:
+            tableName: user_role
+            columns:
+              - column:
```

Comment:
Add constraints to this table. Primary key (user_id, role_id) and foreign keys for both columns